app/uk/gov/hmrc/economiccrimelevyreturns/views/NotRegisteredView.scala.html

```diff
@@ -0,0 +1,30 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.economiccrimelevyreturns.config.AppConfig
+
+@this(
+        layout: templates.Layout,
+        appConfig: AppConfig,
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(messages("notRegistered.title"))) {
+ <h1 class="govuk-heading-xl">@messages("notRegistered.heading")</h1>
+
+ <p class="govuk-body">@Html(messages("notRegistered.body", appConfig.registrationUrl))</p>
```

Comment:
@peter-hazell 

this is probably the way you were imagining the dynamic HTML link to be generated as.